src/detection/card/content/items/attachments.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { Box, Typography } from '@material-ui/core'
+import DescriptionIcon from '@material-ui/icons/Description'
```

Comment:
![image](https://user-images.githubusercontent.com/23569938/125787137-3e845a78-67ce-470d-afc7-03b168a27210.png)
We can use a named import here by doing this.